### ISSUE TITLE
autoconf fails to build (at least on x86_64)

### ISSUE THREAD

**Issue by waddlesplash** [haikuports/haikuports#1472]
```
autoconf.texi:8017: misplaced }
autoconf.texi:8018: must be after `@defmac' to use `@defmacx'
autoconf.texi:8019: misplaced }
autoconf.texi:8206: must be after `@defmac' to use `@defmacx'
autoconf.texi:8271: misplaced }
autoconf.texi:8290: misplaced }
autoconf.texi:8317: misplaced }
autoconf.texi:8380: must be after `@defmac' to use `@defmacx'
conftest.c:4597: must be after `@defmac' to use `@defmacx'
conftest.c:15929: must be after `@defmac' to use `@defmacx'
Makefile:251: recipe for target 'autoconf.info' failed
```
full log at http://vmpkg.haiku-os.org/kitchen/#/build/16

**Comment by diversys** [comment]
This was fixed by upstream in http://git.savannah.gnu.org/gitweb/?p=autoconf.git;a=patch;h=a357718b0
We should probably reapply this patch

**Comment by diversys** [comment]
Should be fixed in d1caeaf.
